book/website/communication/social-media.md

```diff
@@ -20,40 +20,40 @@ Although, maintaining a social media account successfully can be time-consuming.
-There are similar benefits to using Academia.edu or Mendeley, although they are not used as much as Researchgate.
+There are similar benefits to using Academia.edu or Mendeley, although they are not used as much as ResearchGate.
 
 A regular criticism of these sites is the spamming of your email with constant updates that are hard to turn off.
 
-More general social media sites are very popular with academic audiences and a recent Nature survey {cite:ps}`Noorden2014social` highlighted that researchers mostly use Twitter and Linkedin for professional use rather than Facebook.
+More general social media sites are very popular with academic audiences and a Nature survey in 2014 {cite:ps}`Noorden2014social` highlighted that researchers mostly use X and LinkedIn for professional use rather than Facebook.
 
-This survey also showed that Twitter had a wider range of uses compared to the other social media platforms, such as following discussions, posting work content, discovering peers, discovering recommended papers and commenting on others research.
+This survey also showed that X had a wider range of uses compared to the other social media platforms, such as following discussions, posting work content, discovering peers, discovering recommended papers and commenting on others research.
 It has become the communication channel of choice for many researchers and is opening up research discussions at a much earlier stage than previously seen.
 
-Twitter can be used to build up a personal academic profile, for use throughout a research project and can also be used by organisations to promote their work and message.
-It can make research more visible and understandable to wider communities by allowing non-scientists to find new research instantaneously.
+X can be used to build up a personal academic profile, for use throughout a research project and can also be used by organisations to promote their work and message.
 Scientists can also be contacted directly by the public, which enables a more approachable level of communication {cite:ps}`Kelesidou2021scicomm`.
+However recent changes to the platform have also alienated some academics and in the longer term this may precipitate a more widespread move by researchers away from X to more open platforms {cite:ps}`VidalValero2023twitter`.
 
-Due to its popularity and wide range of uses for academic purposes, the next two sub-chapters focus on using Twitter.
-However, below is an overview of the most popular general social media sites used by researchers - Twitter, Linkedin, Facebook and Instagram.
+Despite this X remains popular and given its wide ranging uses for academic purposes the next two sub-chapters focus on how to use it effectively.
+Below we also offer a brief overview of the most popular general social media sites used by researchers - X, LinkedIn, Facebook and Instagram.
 
-### Twitter
+### X (formerly Twitter)
 
 * They have 353 million monthly active users.
```

Comment:
I wonder if we should delete this? Number of active users has fallen dramatically and this figure is almost definitely out of date now

Reply:
I agree that it would be better to remove it because it will indeed go out of date quickly!

Reply:
I tried to find a more recent number so that this could be updated, but there seems to be some disagreement about whether this number is increasing or decreasing, and X may have [changed how it counts](https://variety.com/2023/digital/news/musk-twitter-x-acquisition-one-year-user-revenue-decline-1235770297/) this recently, so it's all a bit messy.

One reason we might want to keep it is to allow comparison with the other platforms listed on the page, so that their relative sizes can still be compared. Would it make sense to keep it, but making it more vague ("Around 300 million..." or "Several hundred million...")?

Reply:
I'd be interested to hear from others on this (@KirstieJane?) 

On the one hand, I agree it's a useful comparison if we can be confident in the ballpark figure, on the other, the level of uncertainty around it maybe makes it less useful? I'd also argue that in all these cases, the total number of users is less important for researchers than whether their target groups/communities are using the platform, but that's definitely not a _lightly_ opinionated take!

Reply:
I have almost no opinion on this but if asked for one I'd say keep the number and put a date after it .

Keeping that number up to date isn't important but some sense of scale is important I think.

Reply:
Thanks for this discussion; this has been really helpful. I've updated the numbers to the latest I could find and added dates into the text so that they're grounded. I've also added in references for them. Maybe these should just be links though?

These numbers came from this [Wikipedia page](https://en.wikipedia.org/w/index.php?title=List_of_social_platforms_with_at_least_100_million_active_users&oldid=1185160092). The references are also from this page; I checked that the links work and state what's claimed.

I'm still more than happy to remove all reference to these numbers if that's the preferred option.

Reply:
ah no I have many hills on which I would die, but this is not one of them 😄 let's keep it in and add the dates like you suggested so that it's contextualised for people